libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -683,6 +684,76 @@ class MapboxNavigation(
         )
     }
 
+    /**
+     * Create and cache a user feedback about an issue or problem with the Navigation SDK.
+     *
+     * Instead of sending the feedback, this API will cache the feedback in the Navigation SDK. The cached feedbacks can be retrieved by calling [getCachedUserFeedback] so the user can update and send them in a later time. To send the cached feedbacks, please use [postCachedUserFeedback].
```

Comment:
We should be explicitly and add a note stating that this is the case only when in _Active Guidance_ (Feedback events are only handled in that mode) and as long as `postCachedUserFeedback` is called before transitioning between modes as that automatically will flush all the pending events. If not, all those events will be lost (as they're not push, only cached). BTW is that the behavior we want for such a use case? Have we tested this scenario? Wondering if we could be smarter when it comes to losing events 🤔 

Refs. https://github.com/mapbox/mapbox-navigation-android/pull/3724#discussion_r515205044